randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1752,10 +1752,10 @@ Extra - asset_id: collision_camera_020
       Morph Ball and After Dairon - Cross Bomb Puzzle Grapple Block
   > Pickup (Missile Tank)
       All of the following:
-          Missiles ≥ 4 and Lay Bomb
+          Missiles ≥ 4 and Lay Bomb and Shoot Missile
           Any of the following:
               Lay Cross Comb
-              Movement (Beginner) and Ballspark
+              Speedbooster Conservation (Beginner) and Ballspark
```

Comment:
let's make it intermediate just to be safe

Reply:
oh i didnt even see this til now